### PR TITLE
Osm reprojection and clipping fixes

### PR DESCRIPTION
**Pull request by RBBentley**
This resolves some issues with OpenStreetMap building display.
1) A model with a terrain bias was not being applied correctly to the reality model
2) Section Clipping was being applied to the OSM reality model.  It is now ignored as with background maps
3) The culling was not handling the reprojection transform.

### CODE REVIEW

File: core/frontend/src/tile/RealityTile.ts [hunk 1]
```diff
@@ -266,14 +270,21 @@ export class RealityTile extends Tile {
     else
       Frustum.fromRange(this.range, scratchFrustum);
 
-    if (this.isFrustumCulled(scratchFrustum, args, true, this.boundingSphere))
+    let boundingSphere = this.boundingSphere;
+
+    if (this._reprojectionTransform) {
+      scratchFrustum.multiply(this._reprojectionTransform);
+      boundingSphere = boundingSphere.transformBy(this._reprojectionTransform, scratchSphere);
```
Comment (pmconne) [inline core/frontend/src/tile/RealityTile.ts:277]: Are you concerned that any other code that accesses the `boundingSphere` property (defined in Tile) will not have this reprojection applied to it?
Did you consider changing `Tile.boundingSphere` to a getter and overriding it in RealityTile to apply the reprojection?

Comment (pmconne) [reply core/frontend/src/tile/RealityTile.ts:277]: Can you supply a test case? I'm concerned that the culling we do for readPixels will not account for the reprojection transform.

Comment (RBBentley) [reply core/frontend/src/tile/RealityTile.ts]: I have refined this such that the reprojectionTransform is applied to the tile range, corners and bounding sphere.   I was only retaining unprojected values for the additive refinement, but I have reworked that to work from original boundary region.   All simpler and more robust now.

Comment (RBBentley) [reply core/frontend/src/tile/RealityTile.ts]: > Are you concerned that any other code that accesses the `boundingSphere` property (defined in Tile) will not have this reprojection applied to it?
> Did you consider changing `Tile.boundingSphere` to a getter and overriding it in RealityTile to apply the reprojection?

I have simplified this such that the reprojection is applied directly to the stored ranges.

Comment (RBBentley) [reply core/frontend/src/tile/RealityTile.ts]: > Are you concerned that any other code that accesses the `boundingSphere` property (defined in Tile) will not have this reprojection applied to it?
> Did you consider changing `Tile.boundingSphere` to a getter and overriding it in RealityTile to apply the reprojection?

This is resolved -- unnecessary with my changes as the boundingSphere is transformed to reflect the reprojection.

**Review by pmconne — APPROVED** [review]

**Comment by pmconne** [comment]
@RBBentley did you queue an ImageTests run? @DStradley has quite a few test cases that display OSM buildings (he might consider making a category for those so they can be run as a subset).

**Comment by pmconne** [comment]
@Mergifyio update

**Comment by calebmshafer** [comment]
@Mergifyio backport release/2.19.x

**Pull request merged**